dxr/hgext/previous_revisions.py

```diff
@@ -0,0 +1,21 @@
+"""previous_revisions
```

Comment:
We should add an addendum to the top-level LICENSE file, along the lines of "The file such-and-such is under the GPL version 2, in conformance with Mercurial's license." And then we'll need the text of the GPL2 after that. You can probably find a split-licensed product somewhere to crib language from.